pyproject.toml

```diff
@@ -0,0 +1,23 @@
+[build-system]
+requires = ["setuptools >= 61.0"]
+build-backend = "setuptools.build_meta"
+
+[project.scripts]
+sherlock = "sherlock:main"
+
+[project.urls]
+Homepage = "http://sherlock-project.github.io/"
+Repository = "https://github.com/sherlock-project/sherlock.git"
+Issues = "https://github.com/sherlock-project/sherlock/issues"
+
+[project]
+name = "sherlock"
+readme = "README.md"
+license = {file = "LICENSE"}
+dynamic = ["dependencies", "version"]
```

Comment:
The version doesnt seem to be found. Would it be better to hard code it using `version = 0.14.3`
```
$ pipx install .         
  installed package sherlock 0.0.0, installed using Python 3.11.6
  These apps are now globally available
    - sherlock
done! ✨ 🌟 ✨
```